src/components/gutter/gutter.ts

```diff
@@ -110,6 +119,63 @@ export class Gutter {
   async layoutItems () {
     if (this.layout) {
       this.layout.layout([...this.items()]);
+      this.layoutComplete.emit();
+    }
+  }
+
+  /**
+   * Method that activates the next item of current active item. The next item's prop `active` will be true. Returns promise with
+   * next active item. If there is no item active,the first item is active. If the current active item is
+   * the last item in the gutter, then when activateNextItem is called, the last item will be active
+   */
+  @Method()
+  async activateNextItem () {
+    if (this.layout) {
+      const items = this.layout.getSortedItems([...this.items()]);
+      const activeItemIndex = items.findIndex(item => item.active);
+      items.forEach(item => { item.active = false; });
+
+      const nextActiveItem = (() => {
+        // if no gutter item is active or if last gutter item is active then go to first gutter item
+        if (activeItemIndex === -1 || activeItemIndex === items.length - 1) {
+          return items[0];
+        } else {
+          // Otherwise go to next gutter item
+          return items[activeItemIndex + 1];
+        }
+      })();
+
+      nextActiveItem.active = true;
+      return nextActiveItem;
+    }
+  }
+
+  /**
+   * Method that activates the previous item of current active item. The next item's prop `active` will be true. Returns promise with
+   * next active item. If there is no item active,the first item is active. If the current active item is
+   * the first item in the gutter, then when activateNextItem is called, the last item will be active
+   */
+  @Method()
+  async activatePrevItem () {
+    if (this.layout) {
+      const items = this.layout.getSortedItems([...this.items()]);
+      const activeItemIndex = items.findIndex(item => item.active);
+      items.forEach(item => { item.active = false; });
+
+      const nextActiveItem = (() => {
```

Comment:
I think it would be more readable to write this as a set of if-else statements. There's no harm in declaring a variable and then using a bunch of if statements to set its value.

Reply:
done